src/main/java/CalculatorApplication.java

```diff
@@ -0,0 +1,64 @@
+import domain.calculator.Calculator;
+import domain.calculator.exception.InvalidInputException;
+import domain.expression.operator.exception.DivideByZeroException;
+import ui.printer.ConsolePrinter;
+import ui.printer.Printer;
+import ui.receiver.ConsoleReceiver;
+import ui.receiver.Receiver;
+
+import java.util.Optional;
+
+public class CalculatorApplication {
+    private static final boolean CONTINUE_PROGRAM = true;
+    private static final String EXIT_REQUEST = "exit";
+
+    private Printer printer;
+    private Receiver receiver;
+    private Calculator calculator;
```

Comment:
final을 추가해주어 재할당을 막아줘도 좋을것 같아요.

```suggestion
    private final Printer printer;
    private final Receiver receiver;
    private final Calculator calculator;
```